recipe/meta.yaml

```diff
@@ -86,4 +86,4 @@ extra:
   skip-lints:
     - host_section_needs_exact_pinnings
     - python_build_tool_in_run
-    - documentation_specifies_language
\ No newline at end of file
+    - documentation_specifies_language
```

Comment:
```suggestion
```

Should be able to just adjust doc_url to https://pytensor.readthedocs.io rather than skipping the lint.